terraform/rds/main.tf

```diff
@@ -18,7 +18,7 @@ resource "aws_rds_cluster_instance" "cluster_instances" {
     count                  = 1
     identifier             = "${var.project_name}-rds-instance"
     cluster_identifier     = aws_rds_cluster.rdsclu.id
-    instance_class         = "db.r4.large" # this is the smallest instance can be use for the engine
+    instance_class         = "db.t2.small" # this is the smallest instance can be use for the engine
```

Comment:
Have you tested it? I tested it before, it was saying t2.samll is not supported for aurodb.

Reply:
yep! it works for me